client/app/shared/recordings/model.js

```diff
@@ -10,7 +10,11 @@ export default Model.extend({
     url: {
       type: 'Text',
       validators: ['url', 'required'],
-    }
+    },
+    organization_id: {
+      type: 'Text',
```

Comment:
Nice. I was kind of imagining that we would write a custom editor (like the WYSYWIG editor) that could edit fields of type "Organization", complete with auto-complete, configuration, etc.